supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc

```diff
@@ -168,6 +168,26 @@ If referring to the program, use "Emacs", for example, "Source-Navigator support
 
 *See also*:
 
+[[executable]]
+==== image:images/yes.png[yes] executable (adjective)
+*Description*: _Executable_ describes a file an operating system can run as a program. For directories, having executable permission allows you to navigate into the directory with the `cd` command.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[[executable]]
+==== image:images/caution.png[with caution] executable (noun)
+*Description*: When it runs, an _executable_ performs specific operations based on its coded instructions. Write _executable file (executable)_ on the first use.
+
+*Use it*: with caution
+
+*Incorrect forms*: "executable" alone on first use
```

Comment:
Our incorrect forms field is usually just a single term with no additional guidance. I'd say to just remove this